arches_component_lab/src/arches_component_lab/widgets/DatePickerWidget/components/DatePickerWidgetEditor.vue

```diff
@@ -26,37 +26,52 @@ const dateFormat = ref();
 watchEffect(() => {
     const convertedDateFormat =
         convertISO8601DatetimeFormatToPrimevueDatetimeFormat(
-            cardXNodeXWidgetData.node.config.dateFormat,
+            props.cardXNodeXWidgetData.node.config.dateFormat,
         );
 
     dateFormat.value = convertedDateFormat.dateFormat;
     shouldShowTime.value = convertedDateFormat.shouldShowTime;
 });
 
-function resolver(event: FormFieldResolverOptions) {
-    if (!event.value) {
-        return null;
-    }
+function transformValueForForm(event: FormFieldResolverOptions) {
+    const date = new Date(event.value);
+
+    try {
+        const formattedDate = formatDate(
+            date,
+            props.cardXNodeXWidgetData.node.config.dateFormat,
+        );
 
-    return formatDate(event.value, cardXNodeXWidgetData.node.config.dateFormat);
+        return {
+            display_value: formattedDate,
+            node_value: formattedDate,
+            details: [],
+        };
+    } catch (_error) {
+        return {
+            display_value: event.value,
+            node_value: event.value,
+            details: [],
+        };
+    }
 }
 </script>
 
 <template>
     <GenericFormField
         v-bind="$attrs"
         :node-alias="nodeAlias"
-        :initial-value="new Date(value?.node_value!)"
-        :resolver="resolver"
+        :transform-value="transformValueForForm"
     >
         <DatePicker
             icon-display="input"
             :date-format="dateFormat"
             :fluid="true"
+            :manual-input="false"
+            :model-value="value.node_value as unknown as Date"
```

Comment:
need to handle for "null" value
```suggestion
            :model-value="value?.node_value as unknown as Date"
```